From c20c736d7bc6ade40851d3f5d691a5a203c42a46 Mon Sep 17 00:00:00 2001
From: jkreierh <jkreierh@90b65887-3827-0410-9a23-83215b262276>
Date: Fri, 13 Jan 2023 12:37:46 +0000
Subject: [PATCH 0996/1000] MANTIS 63620 Mantis 63620: Firealarm handling in
 DTM ConveyorDev.idl version 1.22 New value ERR_FIREALARM_MODE in errorType
 enumeration

git-svn-id: svn://localhost/SelfServiceCommon/trunk@1758 90b65887-3827-0410-9a23-83215b262276
---
 SelfServiceCommon/Massai/idl/Massai/ConveyorDev.idl | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/SelfServiceCommon/Massai/idl/Massai/ConveyorDev.idl b/SelfServiceCommon/Massai/idl/Massai/ConveyorDev.idl
index 55f07154..dd7b8e2c 100644
--- a/SelfServiceCommon/Massai/idl/Massai/ConveyorDev.idl
+++ b/SelfServiceCommon/Massai/idl/Massai/ConveyorDev.idl
@@ -39,7 +39,7 @@
 **/
 module conveyordev
 {
-   const string interface_version = "1.21";
+   const string interface_version = "1.22";
 /*
 using DeviceComponents::indicationListener and DeviceComponents::Generic from mdci.idl
   RC indicationListener::indication(in any ind);
@@ -200,7 +200,8 @@ using DeviceComponents::indicationListener and DeviceComponents::Generic from md
       ERR_INITIALIZING,             /**< In transition to a stable state */
       ERR_MAINTENANCE_MODE,         /**< conveyor is manually switched off for maintenance purpose */
       ERR_FORCEBAG_MODE,            /**< floorwalker can inject bag without checks */
-      ERR_OFF_MODE                  /**< conveyor is manually switched offline */
+      ERR_OFF_MODE,                 /**< conveyor is manually switched offline */
+      ERR_FIREALARM_MODE            /**< firealarm active in SBD area, switch screen dark */
    };
 
 
-- 
2.41.0.windows.1

